src/client/java/README.md

```diff
@@ -0,0 +1,61 @@
+## Description
+This module is DAOS DFS Java API and DAOS DFS implementation of Hadoop FileSystem. There are two submodules, 
+daos-java-api and hadoop-daos.
+
+### daos-java-api
+It wraps most of common APIs from daos_fs.h, as well as some pool creation related APIs from daos_mgmt.h and some pool 
```

Comment:
The original purpose to create pool and container by ourselves is for CI which makes everything from scratch. Later, we re-thought it that we may hook the Java build to DAOS build. We have a task listed to try it.

Reply:
i just meant only for pool creation since it's a management operation. it's not a big deal, just a thought.
for container yes it's good to use the API.